queries.sql

```diff
@@ -64,3 +64,6 @@ where vets.name = 'Maisy Smith'
 group by species.name
 order by count(*) desc;
 
+explain analyze SELECT COUNT(*) FROM visits where animals_id = 4;
+explain analyze SELECT * FROM visits where vets_id = 2;
```

Comment:
- [ ] It would be better if the first 2 queries in the after screenshot shows `Index scan` instead of the `Seq scan and filter` we have currently. Here is a quick fix, kindly remove the indexes and add them once again, after that let us kindly record the screenshot again and see if it changed to the `Index scan`

Reply:
Thank you, I did that.